test/units/testsuite-21.sh

```diff
@@ -8,22 +8,16 @@ set -o pipefail
 systemctl list-jobs | grep -F 'end.service' && SHUTDOWN_AT_EXIT=1 || SHUTDOWN_AT_EXIT=0
 
 at_exit() {
-    # "Safety net" - check for any coredumps which might have not caused dfuzzer
-    # to stop & return an error (we need to do this now before truncating the
-    # journal)
-    # TODO: check fo ASan/UBSan errors
-    local found_cd=0
-    while read -r exe; do
-        coredumctl info "$exe"
-        found_cd=1
-    done < <(coredumpctl -F COREDUMP_EXE | sort -u)
-    [[ $found_cd -eq 0 ]] || exit 1
-
-    # We have to call the end.service explicitly even if it's specified on
+    set +e
```

Comment:
I think either this or `|| :` below could be removed. I'd keep `|| :` and drop `set +e`

Reply:
It can be done later though. I'll go ahead and merge the PR to make it possible to finally turn the test on.